app/assets/javascripts/analytics/static-tracker.js

```diff
@@ -111,6 +111,10 @@
     this.tracker.trackShare(network);
   };
 
+  StaticTracker.prototype.addLinkedDomain = function(trackerId, name, domain) {
```

Comment:
On the fence about that, I thought I'd make the public method shorter. Will update for parity.